File: generated-sources/scala-gatling/mojang-api/src/gatling/scala/org/openapitools/client/model/SecurityChallenge.scala

package org.openapitools.client.model


case class SecurityChallenge (
    _question: SecurityQuestion,
    _answer: SecurityAnswerId
)
object SecurityChallenge {
    def toStringBody(var_question: Object, var_answer: Object) =
        s"""
        | {
        | "question":$var_question,"answer":$var_answer
        | }
        """.stripMargin
}

File: generated-sources/kotlin-spring/mojang-api/src/main/kotlin/org/openapitools/model/SecurityChallenge.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.SecurityAnswerId
import org.openapitools.model.SecurityQuestion
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * Binds a question to an answer id, the answer must respect the answer id.
 * @param question 
 * @param answer 
 */
data class SecurityChallenge (

        @get:NotNull 
        @JsonProperty("question") val question: SecurityQuestion,

        @get:NotNull 
        @JsonProperty("answer") val answer: SecurityAnswerId
) {

}

